server/index.js

```diff
@@ -1,51 +1,51 @@
-const express = require('express');
-const parser = require('body-parser');
-const path = require('path');
-const db = require('./database');
-const cors = require('cors');
-const compression = require('compression');
+// const express = require('express');
+// const parser = require('body-parser');
+// const path = require('path');
+// const db = require('./database');
+// const cors = require('cors');
+// const compression = require('compression');
```

Comment:
Is there a reason you're keeping this file here if everything is commented out? My suggestion is to either delete it or make a comment explaining that you're keeping it for educational purposes. Optional though!

Reply:
good q - I commented it out to make debugging my data-creation file a lil easier, just forgot I left it commented out